### DISCUSSION TITLE
Python and Machine Learning Content for MindsDB contributors

### DISCUSSION THREAD

**Discussion by MichaelLantz** [mindsdb/mindsdb#1776] [General]
Greetings all,

I just want to share what appears to be a great [resource](https://www.humblebundle.com/books/python-programming-oreilly-books?hmb_source=humble_home&hmb_medium=product_tile&hmb_campaign=mosaic_section_4_layout_index_2_layout_type_carousel_tile_index_1_c_pythonprogrammingoreilly_bookbundle) for learning about Python / Machine Learning while also supporting a charity (Code for America).

I'm by no means affiliated with anyone involved in setting up this bundle nor am I making any kind of commission for those who choose to donate. I'm just a big fan of MindsDB and Humble Bundle.

Cheers!

-Michael

**Comment by ZoranPandovski** [comment]
This is great. Thanks for sharing @MichaelLantz :raised_hands:
